packages/app/package.json

```diff
@@ -190,137 +185,111 @@
     "cross-env": "^7.0.3",
-    "crypto-js": "^4.1.1",
     "css-loader": "^6.7.1",
-    "devtron": "^1.4.0",
     "dotenv": "^16.0.2",
     "elasticlunr": "^0.9.5",
-    "electron": "^8.5.5",
+    "electron": "^26.0.0",
     "electron-better-ipc": "^2.0.1",
     "electron-builder": "^23.3.3",
-    "electron-debug": "^3.2.0",
     "electron-devtools-installer": "^3.2.0",
     "electron-localshortcut": "^3.2.1",
-    "electron-log": "^4.4.8",
     "electron-notarize": "^1.2.1",
-    "electron-publisher-s3": "^20.17.2",
-    "electron-store": "^8.0.2",
+    "electron-store": "^8.0.1",
     "electron-updater": "4.3.9",
-    "extract-text-webpack-plugin": "^3.0.2",
     "file-loader": "^6.2.0",
     "fork-ts-checker-webpack-plugin": "^7.2.13",
-    "form-data": "^4.0.0",
-    "fs": "0.0.2",
     "fs-extra": "^10.1.0",
     "getmac": "^5.20.0",
-    "glob": "^8.0.3",
     "history": "^4.10.1",
     "html-webpack-plugin": "^5.5.0",
     "ical-generator": "^3.5.1",
     "ical-js-parser": "^0.7.0",
     "identity-obj-proxy": "^3.0.0",
-    "image-webpack-loader": "^8.1.0",
     "is-electron-renderer": "^2.0.1",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "jest-styled-components": "^7.0.8",
-    "json-loader": "^0.5.7",
     "lodash": "^4.17.21",
     "memfs": "^3.4.7",
     "mini-css-extract-plugin": "^2.6.1",
+    "mock-fs": "^5.2.0",
     "mockdate": "^3.0.5",
     "moment": "^2.29.4",
-    "moxios": "^0.4.0",
     "node-polyfill-webpack-plugin": "^2.0.1",
     "nodemon": "^2.0.19",
     "p-queue": "^7.3.0",
     "quoted-printable": "^1.0.1",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
     "react-beautiful-dnd": "^13.1.1",
-    "react-beautiful-dnd-test-utils": "^4.1.1",
-    "react-dom": "^17.0.2",
+    "react-dom": "^18.2.0",
     "react-google-button": "^0.7.2",
-    "react-hook-form": "~7.12.2",
-    "react-hot-loader": "^4.13.0",
+    "react-hook-form": "~7.47.0",
     "react-intersection-observer": "^9.4.0",
-    "react-intl": "5.0.2",
-    "react-intl-translations-manager": "^5.0.3",
+    "react-intl": "6.4.7",
     "react-modal": "^3.15.1",
-    "react-redux": "^8.0.2",
+    "react-redux": "8.1.3",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-svg-loader": "^3.0.3",
     "react-syntax-highlighter": "^15.5.0",
-    "react-test-renderer": "^17.0.2",
     "react-tooltip": "^4.2.21",
     "react-viewport-list": "^6.0.1",
     "react-virtualized": "^9.22.3",
-    "redux": "^4.2.0",
-    "redux-devtools-extension": "^2.13.9",
+    "react-virtuoso": "^2.18.0",
+    "redux": "4.2.1",
     "redux-logger": "^3.0.6",
     "redux-mock-store": "^1.5.4",
-    "redux-observable": "^2.0.0",
-    "redux-thunk": "^2.4.1",
+    "redux-thunk": "2.4.2",
     "reflect-metadata": "^0.1.13",
+    "reselect": "4.0.0",
     "rrule": "^2.7.1",
-    "sass-loader": "^13.0.2",
-    "semver": "^7.3.7",
-    "source-map-loader": "^3.0.1",
+    "sass-loader": "^12.4.0",
```

Comment:
why was this downgraded?

Reply:
https://github.com/mudita/mudita-center/pull/1550/commits/c91d78ac7c8639525a774875d2894e91d00f0450